tapdb/universe_federation.go

```diff
@@ -421,6 +418,13 @@ func (u *UniverseFederationDB) QueryFederationSyncConfigs(
 		return nil, nil, err
 	}
 
+	// TODO(jhb): Should we also sort by asset ID?
+	// Return config options in a semi-stable order.
+	sort.SliceStable(uniConfigs, func(i, j int) bool {
```

Comment:
Why not just an `ORDER BY` clause? That's why we do SQL in the first place, isn't it?

Reply:
Looks like the order is still different on Postgres...

Reply:
Fixed, wasn't backend-specific but removing the sort after calling `maps.Values()` for the global config.